buildSrc/src/main/kotlin/Project.kt

```diff
@@ -12,6 +12,11 @@ object AndroidVersions {
     const val targetSdkVersion = 30
     const val compileSdkVersion = 31
   }
+  object ExampleApp {
```

Comment:
Should we migrate everywhere to target 31 if Google Play now enforces it?

Reply:
for sdk we can keep as it is, since its upto developers who uses maps-sdk to use targetSdk as 31 for app to deploy. but we can definitely update the downstream app's target version to 31. I will update those.

Reply:
Not sure I understand - why do we want to keep outdated target for the SDK? I assume we might need to update some parts of the SDK as well to correctly handle and adopt all of the requirements of the newer target. 
https://developer.android.com/google/play/requirements/target-sdk from the list of requirements it seems that we need to make sure `location permission` and `intent filters` sections are correctly implemented in the SDK, otherwise users might have problems I assume.

Reply:
> I assume we might need to update some parts of the SDK as well to correctly handle and adopt all of the requirements of the newer target.

yeah, I have this concern only and upgrading target of sdk is not in scope of this pr. 
created [ticket](https://mapbox.atlassian.net/browse/MAPSAND-651) to track this